dhall-openapi/openapi-to-dhall/Main.hs

```diff
@@ -174,6 +176,18 @@ parseImport :: String -> Types.Expr -> Dhall.Parser.Parser Dhall.Import
 parseImport _ (Dhall.Note _ (Dhall.Embed l)) = pure l
 parseImport prefix e = fail $ "Expected a Dhall import for " <> prefix <> " not:\n" <> show e
 
+parseNatIntExceptions :: Options.Applicative.ReadM [(String, String)]
+parseNatIntExceptions =
+  Options.Applicative.eitherReader $ \s ->
+    bimap errorBundlePretty id $ result (pack s)
+  where
+    parser = do
+      typ <- some (alphaNumChar)
+      char '.'
+      fld <- some (alphaNumChar)
+      return (typ, fld)
+    result = parse ((Dhall.Parser.unParser parser `sepBy1` char ',') <* eof) "EXCEPTIONS"
```

Comment:
I don't know if the name shows up in the error message or not, but just in case it does, maybe set the option name as the "file" name:

```suggestion
    result = parse ((Dhall.Parser.unParser parser `sepBy1` char ',') <* eof) "--natIntExceptions"
```

Reply:
The usage string now reads like this:

```
Usage: openapi-to-dhall [--skipDuplicates] [--preferNaturalInt] 
                        [--natIntExceptions EXCEPTIONS] [--prefixMap MAPPING] 
                        [--splitPaths SPLITS] FILE [--crd]
```

An error will come out like so:

```
option --natIntExceptions: EXCEPTIONS:1:133:
  |
1 | ContainerStateTerminated.exitCode,ContainerStateTerminated.signal,PodSpec.priority,PriorityClass.valueCustomResourceColumnDefinition.priority
  |                                                                                                                                     ^
unexpected '.'
expecting ',', alphanumeric character, or end of input
```

The "EXCEPTIONS", "MAPPING", etc. are not variable names, but just strings shown in the usage and error messages respectively. I tried putting something more descriptive for the usage one like `type-1.field-1[,type-2.field-2]+` but it didn't look right and duplicates the `--help` output. I also tried putting something more generic like "ARG" for the error message one but that didn't improve things enough to be worth breaking the connection to the usage string. And duplicating the argument name obviously doesn't make sense. I'm open for any other suggestions but maybe it is not the worst as it is.

Reply:
Ah, okay, I didn't realize that the option name would already show up in the error message.  Then I think you were right to keep it the way it is